pkg/controller/egress_router/egress_router_controller.go

```diff
@@ -0,0 +1,159 @@
+package egress_router
+
+// egress router imnplements a controller for the egress router CNI plugin
+
+import (
+	"context"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/certrotation"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"reflect"
+	"log"
```

Comment:
Use klog for consistency.

Reply:
will change all logs to klog overall, thanks!